hw/bsp/bsp.h

```diff
@@ -30,6 +30,44 @@
 #include <stdbool.h>
 
 #include "board_config.h"
+#include "pinning_options.h"
+
+#ifndef VOLUME_LABEL
```

Comment:
In the samd port these options are in the uf2.h file so I'm a bit torn on where they should go, but I guess seeing as they are board-specific, we might as well have them here

Reply:
I thought about this a lot and took the view that they're potentially board specific, so belong here.  Consider the case that someone wants to use tinyuf2 but have differentiated naming for _their_ board - this allows them to do that.  Yes, I know that breaks universal look and feel, but folks _will_ want change, and if that change isn't supported by the codebase (even through gritted teeth) then they will fork, and that would be worse.